CONTRIBUTING.md

```diff
@@ -28,6 +28,7 @@ Clone the IHP repository into the project directory. The `IHP` directory is adde
 git clone git@github.com:digitallyinduced/ihp.git IHP
 # only needs to be run once (do not run inside the IHP directory)
 make -B build/ihp-lib
```

Comment:
The `direnv allow` needs to happen before the `make -B build/ihp-lib`

Reply:
I think it worked for me also after, but sure - I'll change to be on the safe side